app/components/live-blog/live-blog-api/live-blog-api.js

```diff
@@ -12,9 +12,17 @@ angular.module('bulbs.cms.liveBlog.api', [
 
       var liveBlogEndpoint = CmsConfig.buildApiUrlRoot.bind(null, 'liveblog');
       var liveBlogEntryEndpoint = liveBlogEndpoint.bind(null, 'entry');
+      var liveBlogEntryResponseEndpoint = function (entryId) {
+        return liveBlogEntryEndpoint.bind(null, entryId, 'responses');
+      };
 
-      var parsePayload = function (payload) {
-        var data = _.cloneDeep(payload);
+      var parseEntryPayload = function (payload) {
+        var data = _.chain(payload)
+          .cloneDeep()
+          .mapKeys(function (value, key) {
+            return Utils.toCamelCase(key);
+          })
+          .value();
```

Comment:
This is a cool pattern. 👍 

In our imaginary unified javascript codebase this utility is generally available everywhere.